huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go

```diff
@@ -98,18 +98,15 @@ func ResourceLoadBalancerV3() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 				ConflictsWith: []string{
-					"iptype", "bandwidth_charge_mode", "bandwidth_size", "sharetype",
+					"iptype", "bandwidth_charge_mode", "bandwidth_size", "sharetype", "bandwidth_id",
```

Comment:
Please change the document together(document line 123).
And check the others.

Reply:
done